README.md

```diff
@@ -55,28 +55,35 @@ UI, ready for you to install.
 The graphical SDK will allow you to quickly build and test simple databox apps. The current version of SDK run by default in `dev` mode. To start the sdk run:
 ```
 ./databox-start sdk
-```        
+```
 The SDK web UI is available at http://127.0.0.1:8086
 
-To stop the SDK run: 
+To stop the SDK run:
 ```
 ./databox-stop sdk
 ```
 
-Databox has a number of platform components, divided into two parts:  Core and User components.
+When you start in development mode only the `core-components` are built from source. If you wish to develop one of the available apps or drivers then you can add them to you local install using:
+
+```
+./databox-install-component driver-os-monitor
+```
+
+This will download and build the code on your machine and upload the Databox manifest to your local app store. You can also use this with your repositories  and forks using:
```

Comment:
Is a local app store guaranteed to be running?

Reply:
For now, it is run in all modes and the container manager UI will query both the local and remote app store. If an image is present in both then a drop-down list is displayed to select the one you want.

Reply:
Ok-- so the answer is "yes" currently? Will do for now, obvs we want to move away from that at some point.